plugins/org.jkiss.dbeaver.registry/src/org/jkiss/dbeaver/registry/DataSourceRegistry.java

```diff
@@ -510,6 +511,15 @@ public void updateAuthProfile(DBAAuthProfile profile) {
         }
     }
 
+    @Override
```

Comment:
In fact it is not "add". It is setAuthProfiles

Reply:
Done.